frontend/public/components/machine.tsx

```diff
@@ -281,7 +282,17 @@ export const MachinePage: React.FC<MachinePageProps> = ({
     namespace,
   });
 
-  const [data, filteredData, onFilterChange] = useListPageFilter(machines);
+  const filters: RowFilter[] = [
```

Comment:
looks much better :)

Can you update `RowFilter` type so it can contain only `type` and `filter` ?

currently we have 
```js
export type RowFilter<R = any> = RowMatchFilter<R> | RowReducerFilter<R>;
```

so we could add something like (maybe with better naming)
```js
type SimpleRowFilter<R> = {
  type: string;
  filter: (input: FilterValue, obj: R) => boolean;
}

export type RowFilter<R = any> = RowMatchFilter<R> | RowReducerFilter<R> | SimpleRowFilter<R>;
```